src/js/veteran-representative/config/form.js

```diff
@@ -0,0 +1,274 @@
+
+const limitationOfConsentDescription =
+  "I authorize disclosure of records related to treatment for all conditions listed in Item 12 except:";
+
+const authorizationToChangeClaimantAddressDescription =
+  "I authorize any official representative of the organization named in Item 3A to act on my behalf to change my address in my VA records. This authorization does not extend to any other organization without my further written consent. This authorization will remain in effect until the earlier of the following events: (1) I file a written revocation with VA; or (2) I appoint another representative, or (3) I have been determined unable to manage my financial affairs and the individual or organization named in Item 3A is not my appointed fiduciary";
+
+const formConfig = {
+  urlPrefix: "/",
+  submitUrl: "/v0/api",
+  trackingPrefix: "form-2122-",
+  introduction: IntroductionPage,
+  confirmation: ConfirmationPage,
+  formId: "21-22",
+  version: 0,
+  prefillEnabled: true,
+  savedFormMessages: {
+    notFound: "Please start over to apply for veteran representative.",
+    noAuth:
+      "Please sign in again to continue your application for veteran representative.",
+  },
+  title: "Appoint a Veteran Service Officer as your representative",
+  defaultDefinitions: {
+    fullName,
+  },
+  chapters: {
+    veteranInformation: {
+      title: "Veteran Information",
+      pages: {
+        veteranInformation: {
+          path: "veteran-information",
+          title: "Veteran information",
+          uiSchema: {
+            fullName: fullNameUI,
+            veteranSSN: ssnUI,
+            vaFileNumber: { "ui:title": "VA file number" },
+            insuranceNumber: { "ui:title": "Insurance Number" },
+          },
+          schema: {
+            type: "object",
+            required: [
+              'fullName',
+              'veteranSSN',
+              'vaFileNumber',
+              'insuranceNumber',
+            ],
+            properties: {
+              fullName,
+              veteranSSN,
+              vaFileNumber,
+              insuranceNumber,
+            },
+          },
+        },
+      },
+    },
+    claimantInformation: {
+      title: "Claimant Information",
+      pages: {
+        claimantInformation: {
+          path: "claimant-information",
+          title: "Claimant Information",
+          uiSchema: {
+            fullName: _.merge(fullNameUI, {
+              first: {
+                "ui:title": "Claimant’s first name",
+              },
+              last: {
+                "ui:title": "Claimant’s last name",
+              },
+              middle: {
+                "ui:title": "Claimant’s middle name",
+              },
+              suffix: {
+                "ui:title": "Claimant’s suffix",
+              },
+            }),
+            // FIX: need a default field for dropdown
```

Comment:
As long as the schema for this field has an `enum` property, it should turn into a drop down automatically.

Reply:
Ahh ok, thank you! If it goes against the grain I won't add it, I wasn't sure what the standard was here.